docs/quickstart.rst

```diff
@@ -1,6 +1,68 @@
 Quickstart
 ==========
 
+Use *septentrion* as your migration tool in 2 easy steps:
+  1. Write your migrations.
+  2. Run your migrations.
+
+Write migrations
+----------------
+Your migration folder should be organised as follow:
+ - each version has its own folder;
+ - in a version folder, migrations will be executed in alphabetical order;
+ - `schemas` folder contains state of the schema at a given version.
+
+# I'm not sure if the explanation around the naming convention makes sense in a quick start, probably not?
+# Do we need something more in that section?
```

Comment:
I guess we should be holding hands much more. Here, we're not helping people implement septentrion for their project, we're taking them on a tour showcasing septentrion. We can put ourselves in a situation where we control everything.  
```
mkdir example
cd example
virtualenv .venv
source .venv/bin/activate
mkdir 1.0
edit 1.0/<name of the first migration>.sql
<spell out exactly what to write in the migration file>
Quickly create 2 or 3 other migrations
```